### DISCUSSION TITLE
H2 support ?

### DISCUSSION THREAD

**Discussion by s-kocher** [FerretDB/FerretDB#729] [General]
I see there is a discussion for MySQL support. Same question for H2, it will be great to have thanks to FerretDB + H2 an in memory MongoDB for unit / integration tests or POC for example.

**Comment by AlekSi** [comment]
In such a scenario, you will still need to run FerretDB as a separate process. So you might as well run a temporary PostgreSQL instance in the same Docker Compose configuration, for example.
